extensions/vertx-http/deployment/src/main/resources/dev-templates/logmanager.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
```

Comment:
That will still end up in the HTML, no? I guess most browsers will ignore it, but probably we should remove it, no?

Reply:
OK, removed :)